### ISSUE TITLE
Title screen video being downscaled...

### ISSUE THREAD

**Issue by aliendiplomat** [RoseTheFlower/UltrawideIndex#58]
Hopefully this is in scope and something you could look at. I use a 3440x1440 screen and notice that videos on the title screen are being scaled down by 50%. This applies to the main game video, but I also tried replacing this with my own custom 2440x1440 bink video and while it plays, it is also scaled down 50%.

It would be great if this modification could correct this. Editing the .EXE with a hex editor to replace 16:9 refs with 3440x1440 does fix the video downscaling, however it completely breaks the rest of the UI.

**Comment by RoseTheFlower** [comment]
Which game?

**Comment by aliendiplomat** [comment]
OMG sorry, I assumed this was just for Starfield. It's Starfield. :)

**Comment by RoseTheFlower** [comment]
The video is in 3840x2160, so is it not natural for it to be downscaled at 1440p?

**Comment by aliendiplomat** [comment]
It would make sense if it was resizing to fit but that isn't actually what is happening. For example, my 3440x1440 video should not resize at all since that is the screen resolution. However it is being sized down so it floats in the middle with black letterboxing for width and height. The same happens with the default video.

**Comment by RoseTheFlower** [comment]
It makes sense that the native 16:9 video is pillarboxed, as the alternative is cropping it to fill. I'm not seeing any quality issues when comparing the video to the main menu at 1440p. It seems you just want to be able to use your own 21:9 video without the game pillarboxing it.

**Comment by aliendiplomat** [comment]
No, the problem is that while the game natively supports 21:9 for rendering, the title screen is still locked at 16:9. That is what I am hoping this can resolve. I would like to see the title screen space extended to the full dimensions set for the game resolution.

That way a 3440x1440 video will play full screen, instead of being resized down to fit a 16:9 screen width.
